bilby/core/utils/random.py

```diff
@@ -11,7 +11,11 @@ class Generator:
 
 
 def seed(seed):
+    from .meta_data import global_meta_data
+
     Generator.rng = default_rng(seed)
+    global_meta_data.add_to_meta_data("rng", Generator.rng)
```

Comment:
I thought we may want to add some form of check/validation when adding values, so this would make it easier in future. Happy to revert to something simpler though